sesman/sesman.c

```diff
@@ -177,110 +282,142 @@ sesman_main_loop(void)
     return rv;
 }
 
+/*****************************************************************************/
+static void
+print_version(void)
+{
+    g_writeln("xrdp-sesman %s", PACKAGE_VERSION);
+    g_writeln("  The XRDP session manager");
```

Comment:
We see both the uppercase XRDP and the lowercase xrdp. There are no explicit rules but there more lowercase forms than uppercase forms. This is really my personal opinion, the lowercase xrdp looks cooler.

No problem, this is not important at all.

Reply:
Let's go with that then.

I've added a [page](/neutrinolabs/xrdp/wiki/Referring-to-the-project) to the wiki  specifying this as the project preference. It's linked from the main page.